modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+
+| store
+a|
+Information about the storage engine and the store format.
+
+The value is a string formatted as:
+
+[source, syntax, role="noheader"]
+----
+{storage engine}-{store format}-{major version}.{minor version}
+----
+| STRING
+
+| lastCommittedTxn
+| The ID of the last transaction received.
+| INTEGER
+
+| replicationLag
+|
+Number of transactions the current database is behind compared to the database on the primary instance.
+The lag is expressed in negative integers. In standalone environments, the value is always `0`.
+| INTEGER
+
+|constituents
+|The names of any constituents the database may have. label:default-output[]
+| LIST OF STRING
+
+|===
+
+
+=== Show all available databases
+
+A summary of all available databases can be displayed using the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES
+----
+
+.Result
+[role="queryresult"]
+----
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name     | type       | aliases                      | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "movies" | "standard" | ["films", "motion pictures"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
+| "neo4j"  | "standard" | []                           | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | TRUE    | TRUE  | []           |
+| "system" | "system"   | []                           | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+The results of this command are filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
+* Users with `CREATE/DROP COMPOSITE DATABASE` or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
+* Users with `DATABASE MANAGEMENT` privilege can see all databases.
+
+If a user has not been granted `ACCESS` privilege to any databases nor any of the above special cases, the command can still be executed but will only return the `system` database, which is always visible.
```

Comment:
Have changed (now both are outside of the list, under the "show all ..." section

Reply:
you have moved them but not addressed what my comment was actually about.

Will clarify my initial comment:

In the alias section we have as the last point in the list of special cases:
```
* Users can always see the `system` database.
```
and then the text under the special cases list says:
```
If a user has not been granted the `ACCESS` privilege to the target database and none of the above special cases apply, then it is not visible to the user and this column will be `null`.
```

While in this section it isn't in the special cases list but instead in the sentence after
```
If a user has not been granted `ACCESS` privilege to any databases nor any of the above special cases, the command can still be executed but will only return the `system` database, which is always visible.
```

This is what I was commenting on, I think I like the alias version better and would therefor change here to add
```
* Users can always see the `system` database.
```
to the special cases list and update the sentence below somehow to match?

Reply:
Also this section should probably be underneath the columns table before the all databases section since it technically applies to all variations of the command 🤷